pkg/varlinkapi/images.go

```diff
@@ -41,13 +41,18 @@ func (i *LibpodAPI) ListImages(call iopodman.VarlinkCall) error {
 	for _, image := range images {
 		labels, _ := image.Labels(getContext())
 		containers, _ := image.Containers()
+		repoDigests, err := image.RepoDigests()
+		if err != nil {
+			return err
```

Comment:
I dunno why everyone else is ignoring errors here, but it's been that way since 39a7a773 (#669).  @baude, do you remember your motivation?

Reply:
`image.Labels(...)` seems to ignore errors as well.

Reply:
> `image.Labels(...)` seems to ignore errors as well.

Yeah, I left the existing error-ignores alone, since I don't understand their motivation.  I'm happy to make these all consistent though (one way or the other) if someone can explain the motivation.

Reply:
It looks like we don't want to fatally fail when trying to get some information on an image, but it's not commented at all so I'm not sure. @baude this intentional?